devel/standard/standard.yaml

```
@@ -1,5 +1,5 @@
 start:
   args:
-  - mindreader-node
+  - reader-node
   flags:
-    mindreader-node-node-boot-nodes: ed25519:58Ra6ybSsjtAo8aHJk65Ed5mfUkVekR2aMBZo5Q8C8ir@127.0.0.1:24560
\ No newline at end of file
+    reader-node-node-boot-nodes: ed25519:58Ra6ybSsjtAo8aHJk65Ed5mfUkVekR2aMBZo5Q8C8ir@127.0.0.1:24560
```

```suggestion
    reader-node-boot-nodes: ed25519:58Ra6ybSsjtAo8aHJk65Ed5mfUkVekR2aMBZo5Q8C8ir@127.0.0.1:24560
```

Seems there was already a typo before.